tools/release/adot-operator-images-mirror/config.yaml

```diff
@@ -36,12 +38,14 @@ sourceRepos:
       - 1.18.0
       - 1.19.1
       - 1.20.2
+      - 1.22.1
```

Comment:
Do we need to mirror the versions in between this? @erichsueh3 can you help with this question.

Reply:
For reference `1.22.0`, and `1.21.0` are not in the mirror list.

Reply:
Nope, we only need to mirror the versions that are used with the Operator versions in the EKS add-on